cime_config/machines/config_machines.xml

```diff
@@ -1503,7 +1503,8 @@
     </module_system>
     <RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR>
     <EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT>
-    <TEST_TPUT_TOLERANCE>0.1</TEST_TPUT_TOLERANCE>
+    <TEST_TPUT_TOLERANCE>0.05</TEST_TPUT_TOLERANCE>
```

Comment:
So, this tolerance would account for system variability as well as changes in a specific PR. I presume 0.05 tolerance translates to 5%. 
Even after turning off output, there may be a few outliers where reads/communication might occasionally trigger a false positive. Hopefully we don't encounter too many of those.